filters/flowid/filter.go

```diff
@@ -54,16 +72,13 @@ func (spec *flowIdSpec) CreateFilter(fc []interface{}) (filters.Filter, error) {
 		} else {
 			return nil, filters.ErrInvalidFilterParameters
 		}
-	}
-	var flowIdLength = defaultLen
-	if len(fc) > 1 {
-		if l, ok := fc[1].(float64); ok && l >= MinLength && l <= MaxLength {
-			flowIdLength = int(l)
-		} else {
-			return nil, filters.ErrInvalidFilterParameters
+		if len(fc) > 1 {
+			log.Println("flow id filter warning: this syntaxt is deprecated and will be removed soon. " +
+				"please check updated docs")
 		}
 	}
-	return &flowId{reuseExisting, flowIdLength}, nil
+	return &flowId{reuseExisting: reuseExisting, generator: spec.generator}, nil
 }
 
-func (spec *flowIdSpec) Name() string { return Name }
+// Name returns the canonical filter name
+func (_ *flowIdSpec) Name() string { return Name }
```

Comment:
I understand and I also spent some time thinking about it. Since this is a relatively easy functionality to deliver, I propose we wait for someone to specifically to ask for it. Until then, we keep the previous functionality of having one single flow ID format per skipper instance.